.gitignore

```diff
@@ -0,0 +1,2 @@
+*.html
```

Comment:
Please don't add `.gitignore` to this repo. It clashes with my (and potentially others') local `.gitignore` file. And if someone decides to generate coverage reports, it should be their own duty to clean them up before checking anything in. (I wouldn't accept pull requests with these files anyway.)

Reply:
Done!